src/control/run_go_tests.sh

```diff
@@ -93,6 +93,9 @@ echo "  CGO_CFLAGS: $CGO_CFLAGS"
 echo "  GOPATH: $GOPATH"
 echo
 
+echo "Checking for libnuma"
+echo $(ldconfig -p | grep libnuma)
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Useless echo? Instead of 'echo $(cmd)', just use 'cmd'. [SC2005]